types/errors/link.go

```diff
@@ -0,0 +1,12 @@
+package errors
+
+const (
+	linkCodespace = "link"
+)
+
+var (
+	// ErrError             = Register(linkCodespace, 1, "error")
```

Comment:
This comment seems to be a bug or test code. 
How about removing it or adding the reason comment above it?

Reply:
OK, I will remove them.